homeassistant/components/aws/image_processing.py

```diff
@@ -49,12 +49,14 @@
 DATETIME_FORMAT = "%Y-%m-%d_%H:%M:%S"
 
 
-def get_valid_filename(name: str) -> str:
+async def get_valid_filename(name):
```

Comment:
type hints always nice to have also

Reply:
Was just going for an all-or-nothing scenario, great for maintainability purposes so will refactor them in just have to focus time on docs first 😄